README.md

```diff
@@ -35,6 +35,15 @@
     conflictstyle = zdiff3
 ```
 
+Or run:
+
+```sh
+$ git config --global core.pager delta
+$ git config --global interactive.diffFilter 'delta --color-only'
+$ git config --global delta.navigate true
+$ git config --global merge.conflictStyle zdiff3
+```
```

Comment:
Hi, thanks for this. Could you remove the pseudo-prompt prefixes so that it's pasteable?

Reply:
Sure thing. I actually prefer without as well, but was working on something else this morning where they were included, so guess that rubbed off